pages/DSNP/Types/GraphChange.md

```diff
@@ -32,48 +30,24 @@ Different change types have different meanings.
 | 0 | Unfollow | Remove a Follow relationship |
 | 1 | Follow | Create a Follow relationship |
 
-### createdAt
-
-- MUST be set to the milliseconds since Unix epoch at time of signing
-- MUST be unique for the given `fromId` and `objectId` pair
-
 ### fromId
 
 - MUST be a [DSNP User Id](../Identifiers.md#dsnp-user-id)
-- MUST be the [signer](../Signatures.md) of the Announcement
+- MUST directly or via a chain of delegation have authorized the creation of the Announcement
 
```

Comment:
How about 

MUST have, directly or via a chain of delegation, authorized the creation of the Announcement

Reply:
Good idea. I'll update.

Reply:
@saraswatpuneet Updated to `MUST have authorized the creation of the Announcement, either directly or via a transparent chain of delegation`. Does that work?

Reply:
@wilwade  yes that looks great thank you